libs/service/verifier_service.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#include <filesystem>
+#include <iostream>
+#include <sstream>
+#include <sys/stat.h>
+#include "api_common.hpp"
+#include "ebpf_api.h"
+#include "ebpf_bind_program_data.h"
+#include "ebpf_platform.h"
+#include "ebpf_xdp_program_data.h"
```

Comment:
Why do you need to include this, and line 12?  I don't see any xdp or bind code in this cpp file.